lib/fhir/resources/practitioner.js

```diff
@@ -11,21 +11,21 @@
 const logger = require('winston')
 
 const FhirCommon = require('../common')
-const constants = require('../../constants')
-const QueryUtils = require('../query-utils')
 const QueryValidator = require('../query-validator')
+const QueryBuilder = require('../query-builder')
 
 module.exports = (mongo) => {
   const fhirCommon = FhirCommon(mongo)
-  const queryUtils = QueryUtils(mongo)
   const queryValidator = QueryValidator()
+  const queryBuilder = QueryBuilder(mongo)
 
   let afterReadHook = (ctx, data, callback) => {
     if (ctx.query && ctx.query._summary) {
       fhirCommon.addSubsettedSecurityLabel(data)
-      delete data.identifier
-      delete data.telecom
       delete data.photo
+      delete data.practitionerRole
```

Comment:
Im not sure how applicable it is, but some of the properties within practitionerRole should be included in the summary data if the parameter is supplied.

https://hl7.org/fhir/DSTU2/practitioner-definitions.html (Practitioner.practitionerRole.role)

Reply:
This property is also DSTU2 specific, and has been moved into its own resource in STU3

Reply:
Oh, I see. Ya we shouldn't remove this it seems.

It is DSTU2 specific, however this code is backward compatible as deleting something that doesn't exist does nothing.